### ISSUE TITLE
DV | Veteran information updates

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#116041]
## Issue Description

The Veteran information page within the Dependents Verification form needs the following changes:

- The "We've prefilled some information for you" Alert header should be an H3 to nest properly on the page
    <img width="700" alt="Veteran information page with HeadingsMap extension showing heading levels" src="https://github.com/user-attachments/assets/babf4e63-bc33-4aa3-8ddb-4a1bb495b775" /> 
- Find all instances that call 21-0538 an "application" and change it to "form" ([ref](https://dsva.slack.com/archives/C05NEL4DKMX/p1754423445635239)). The form config is missing the following settings:

    ```js
    customText: {
      appType: 'form',
      appAction: 'your form',
      continueAppButtonText: 'Continue your form',
      startNewAppButtonText: 'Start a new form',
      appSavedSuccessfullyMessage: 'Your form has been saved.',
      finishAppLaterMessage: 'Finish this form later',
      reviewPageTitle: 'Review your form',
      submitButtonText: 'Submit form',
    };
    ```

## Tasks
- [x] Change prefill alert header to an H3
- [x] Add `customText` to form config
- [x] Find all instances of "application" used in content and change to "form"
- [x] Update unit tests as needed

## Acceptance Criteria
- [x] Veteran info page matches design
- [x] Form 21-0538 is referred to as a "form", not "application"
- [x] All tests passing

**Comment by davidakennedy** [comment]
@Mottie Also this:

Content:

Design says: "This is part of the information we’ll submit on your behalf for your verification of dependents (VA Form 21-0538)."

Staging says: "This is part of the information we’ll submit on your behalf for your verification of dependents (VA Form 21-0538)"

Missing period at end of sentence.

[Figma file](https://www.figma.com/design/bvj72inycD0iZkuCbjYTWL/Dependent-Verification-MVP?node-id=9842-15542&t=AIgQ7xpOB9D7wZMo-4)

Staging:

<img width="585" height="454" alt="Image" src="https://github.com/user-attachments/assets/4aa4b10b-38b3-4f75-bb2f-4deea2959d84" />

Design:

<img width="361" height="200" alt="Image" src="https://github.com/user-attachments/assets/50efa261-7471-4bd0-a7d9-16be896be494" />

Pulling over from #116374.

**Comment by Mottie** [comment]
Updates have been merged and are ready for validation in staging.

**Comment by fiorella-io** [comment]
Looks good! ✅
<img width="30%" src="https://github.com/user-attachments/assets/db7a766d-180f-4325-9665-c24b53c73fc8" />
Ignore the PDF icons - it's my extension issue

@Mottie mentioned that `We’ll save your application on every change.` is hard coded and it might not be able to change it. 

@hnvt1989 The form saved alert issue has been captured here: [#116400](https://github.com/department-of-veterans-affairs/va.gov-team/issues/116400)
